listenbrainz/webserver/templates/index/about.html

```diff
@@ -0,0 +1,108 @@
+{%- extends 'base.html' -%}
+{%- block title -%}Goals - ListenBrainz{%- endblock -%}
+{%- block content -%}
+<h2 class="page-title">Goals</h2>
+
+<p>The ListenBrainz project has a number of goals:</p>
+<ol>
+    <li>
+        A public store of your listen history. We feel that a listen history has a tremendous
```

Comment:
I'll make a tiny suggestion while we're here modifying these files: We could make the first sentence of each item **bold** in this list and the anti-goals.

Here's with simply wrapping them with a `<b>` tag, I think it improves readability:

![image](https://user-images.githubusercontent.com/6179856/153647963-f21ce759-acb0-4554-a4af-8f23db9049a4.png)

Reply:
Done!